### PR TITLE
Ordering openapitools/openapi-generator-cli

### PR DESCRIPTION
**Pull request by pascalgrimaud**
-   Please make sure the below checklist is followed for Pull Requests.

-   [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [ ] Tests are added where necessary
-   [ ] Documentation is added/updated where necessary
-   [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/server/templates/package.json.ejs [hunk 1]
```diff
@@ -36,9 +37,8 @@
     <%_ } _%>
     "prettier": "<%= PRETTIER_VERSION %>",
```
Comment (murdos) [inline generators/server/templates/package.json.ejs:38]: There will be an issue if `prettierJava` is `false`, because of the trailing comma here.

Comment (pascalgrimaud) [reply generators/server/templates/package.json.ejs:38]: oh right, nice catch ! I'm fixing it :D

**Comment by DanielFran** [comment]
@pascalgrimaud Just a minor change, can you update version to 1.0.13-4.3.1 please?

**Comment by pascalgrimaud** [comment]
@avdev4j and @juliensadaoui : are you ok to upgrade to `1.0.13-4.3.1` ? I don't use this lib, and I know you encounter some issue with a specific version but I don't know which one.

**Comment by juliensadaoui** [comment]
there is no problem with minor version

**Pull request merged**